### PR TITLE
Content - Add Proof of Service section

### PR DESCRIPTION
**Pull request by thephez**
- Describe v0.14.0 PoSe scoring

See https://thephez.github.io/en/developer-guide#proof-of-service

### CODE REVIEW

File: _includes/devdoc/guide_dash_features.md [hunk 1]
```diff
@@ -878,3 +878,61 @@ Note the following timeouts defined by Dash Core related to signing sessions:
...
+| Service | Percent of Score | Requirement |
+| ----------- | ---- | ------------------- |
+| LLMQ DKG    | 100% | Participate in the DKG process used to establish LLMQs. Requires exchanging messages with other quorum members. |
+
+**PoSe Score Calculation**
+
+As shown in the following table, the PoSe Score always decreases by 1 per block
+as long as a masternode has not been banned. Once banned, the masternode can
+only be restored by sending a Provider Update Registrar (ProUpRegTx) special
+transaction.
```
Comment (PastaPastaPasta) [inline _includes/devdoc/guide_dash_features.md]: Don't remember this being the case. I thought they auto-unbanned

File: _includes/devdoc/guide_dash_features.md [hunk 2]
```diff
@@ -878,3 +878,61 @@ Note the following timeouts defined by Dash Core related to signing sessions:
...
+
+| PoSe Parameter | Value | Example Value |
+|-|-|-|
+| Maximux PoSe Score | Number of registered masternodes | 5000 |
+| PoSe Score Increases | Maximum PoSe Score * 2/3 | 3333 |
+| PoSe Score Decreases | 1 (per block) | Always `1` |
+
+The current PoSe scoring algorithm increases the PoSe score by 66% for each
```
Comment (PastaPastaPasta) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
The current PoSe scoring algorithm increases the PoSe score by 66% of the number of masternodes for each
```

File: _includes/devdoc/guide_dash_features.md [hunk 3]
```diff
@@ -878,3 +878,61 @@ Note the following timeouts defined by Dash Core related to signing sessions:
...
+failed DKG session. Depending on timing, this allows for 2 (at most) failures
```
Comment (PastaPastaPasta) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
failed DKG session. Depending on timing, this allows for at most 2 failures
```

File: _includes/devdoc/guide_dash_features.md [hunk 4]
```diff
@@ -878,3 +878,61 @@ Note the following timeouts defined by Dash Core related to signing sessions:
...
+for a masternode within a timeframe corresponding to the payment cycle (i.e a
```
Comment (PastaPastaPasta) [inline _includes/devdoc/guide_dash_features.md]:
```suggestion
for a masternode within the payment cycle (i.e a
```

File: _includes/devdoc/guide_dash_features.md (hunk 1 above)
Comment (thephez) [reply _includes/devdoc/guide_dash_features.md]: Good catch. They don't auto-unban, but I have the wrong special tx (it's [ProUpServTx](https://github.com/dashpay/dips/blob/master/dip-0003.md#updating-service-features-from-operator-proupservtx))

**Comment by thephez** [comment]
Thanks, I will take mostly good for now :slightly_smiling_face:

**Pull request merged**